submissions/israelOladosu/DomManipulation/contact.js

```diff
@@ -48,3 +50,13 @@ function validateForm(e) {
 }
 
 submitForm.addEventListener("submit", validateForm);
+
+newsLetter.addEventListener("submit", function () {
```

Comment:
Instead of doing this, just include the `product.js` in the HTML pages that has News Letter

Reply:
Alright... Thank you
Noted.